ansible/roles/ocp4-workload-machinesets/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Do not modify this file
+
+- name: Running Pre Workload Tasks
+  include_tasks:
+    file: ./pre_workload.yml
```

Comment:
you don't need the `./`

Reply:
@fridim Those are in every single workload....

Reply:
OK we can mass-fix later then.